userguide/content/en/docs/adding-content/taxonomy.md

```diff
@@ -180,25 +252,70 @@ Example usage in partial `taxonomy_terms_clouds` for showing all defined taxonom
-    <li><a class="taxonomy-term" href="//localhost:1313/categories/category-2/" data-taxonomy-term="category-2"><span class="taxonomy-label">category 2</span><span class="taxonomy-count">1</span></a></li>
-    <li><a class="taxonomy-term" href="//localhost:1313/categories/category-3/" data-taxonomy-term="category-3"><span class="taxonomy-label">category 3</span><span class="taxonomy-count">2</span></a></li>
-    <li><a class="taxonomy-term" href="//localhost:1313/categories/category-4/" data-taxonomy-term="category-4"><span class="taxonomy-label">category 4</span><span class="taxonomy-count">6</span></a></li>
+    <li>
+      <a
+        class="taxonomy-term"
+        href="//localhost:1313/categories/category-1/"
+        data-taxonomy-term="category-1"
+        ><span class="taxonomy-label">category 1</span
+        ><span class="taxonomy-count">3</span></a
+      >
+    </li>
+    <li>
+      <a
+        class="taxonomy-term"
+        href="//localhost:1313/categories/category-2/"
+        data-taxonomy-term="category-2"
+        ><span class="taxonomy-label">category 2</span
+        ><span class="taxonomy-count">1</span></a
+      >
+    </li>
+    <li>
+      <a
+        class="taxonomy-term"
+        href="//localhost:1313/categories/category-3/"
+        data-taxonomy-term="category-3"
+        ><span class="taxonomy-label">category 3</span
+        ><span class="taxonomy-count">2</span></a
+      >
+    </li>
+    <li>
+      <a
+        class="taxonomy-term"
+        href="//localhost:1313/categories/category-4/"
+        data-taxonomy-term="category-4"
+        ><span class="taxonomy-label">category 4</span
+        ><span class="taxonomy-count">6</span></a
+      >
+    </li>
   </ul>
 </div>
 ```
 
 ### taxonomy_terms_clouds
 
-The partial `taxonomy_terms_clouds` is a wrapper for the partial `taxonomy_terms_cloud` with the only parameter `context` (most of the time the current page or context `.`) and checks the taxonomy parameters of your project's `hugo.toml`/`hugo.yaml`/`hugo.json` to loop threw all listed taxonomies in the parameter `taxonomyCloud` resp. all defined taxonomies of your page, if `taxonomyCloud` isn't set.
+The partial `taxonomy_terms_clouds` is a wrapper for the partial
+`taxonomy_terms_cloud` with the only parameter `context` (most of the time the
+current page or context `.`) and checks the taxonomy parameters of your
+project's config to loop threw all listed taxonomies in the parameter
+`taxonomyCloud` resp. all defined taxonomies of your page, if `taxonomyCloud`
+isn't set.
 
 ## Multi language support for taxonomies
 
-The taxonomy terms associated content gets only counted and linked WITHIN the language! The control parameters for the taxonomy support can also get assigned language specific.
+The taxonomy terms associated content gets only counted and linked WITHIN the
+language! The control parameters for the taxonomy support can also get assigned
+language specific.
```

Comment:
I know this isn't from your change but it feels like there's a word missing from the end of this sentence...

Reply:
Fixed.